packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -340,39 +395,110 @@ const VotePage = ({
               <Card.Body className="p-2">
                 <Card.Text className="py-2 m-0">
-                  <span>Against</span>
-                  <span>{proposal?.againstCount}</span>
+                  <span className={`${classes.voteCardHeaderText} ${classes.against}`}>
+                    Against
+                  </span>
+                  <span className={classes.voteCardVoteCount}>{proposal?.againstCount}</span>
                 </Card.Text>
-                <ProgressBar variant="danger" now={againstPercentage} />
+                <VoteProgresBar
+                  variant={ProgressBarVariant.AGINST}
+                  percentage={againstPercentage}
+                />
+                <Row className={classes.nounProfilePics}>
+                  <NounImageVoteTable nounIds={againstNouns} />
+                </Row>
               </Card.Body>
             </Card>
           </Col>
           <Col lg={4}>
             <Card className={classes.voteCountCard}>
               <Card.Body className="p-2">
                 <Card.Text className="py-2 m-0">
-                  <span>Abstain</span>
-                  <span>{proposal?.abstainCount}</span>
+                  <span className={`${classes.voteCardHeaderText} ${classes.abstain}`}>
+                    Abstain
+                  </span>
+                  <span className={classes.voteCardVoteCount}>{proposal?.abstainCount}</span>
                 </Card.Text>
-                <ProgressBar variant="info" now={abstainPercentage} />
+                <VoteProgresBar
+                  variant={ProgressBarVariant.ABSTAIN}
+                  percentage={abstainPercentage}
+                />
+                <Row className={classes.nounProfilePics}>
+                  <NounImageVoteTable nounIds={abstainNouns} />
+                </Row>
               </Card.Body>
             </Card>
           </Col>
         </Row>
+
+        <Row>
+          <Col lg={4}>
+            <Card className={classes.voteInfoCard}>
+              <Card.Body className="p-2">
+                <Row className={classes.voteMetadataRow}>
+                  <Col className={classes.voteMetadataRowTitle}>
+                    <h1>Threshold</h1>
+                  </Col>
+                  <Col>
+                    <span>Differential</span>
+                    <h3>{proposal.forCount - proposal.againstCount} votes</h3>
+                  </Col>
+                  <Col>
+                    <span>Quorum</span>
+                    <h3>{proposal.quorumVotes} votes</h3>
+                  </Col>
+                </Row>
+              </Card.Body>
+            </Card>
+          </Col>
+          <Col lg={4}>
+            <Card className={classes.voteInfoCard}>
+              <Card.Body className="p-2">
+                <Row className={classes.voteMetadataRow}>
+                  <Col className={classes.voteMetadataRowTitle}>
+                    <h1>{startOrEndTimeCopy()}</h1>
```

Comment:
In order to prevent overflow, let's use the three letter abbreviation for month `MMM`:

<img width="998" alt="1 votes" src="https://user-images.githubusercontent.com/85328329/149664185-1ff04794-2b0b-484c-9d0a-4d7cf84d5e09.png">